test/api_endpoints/test_dbquery_endpoints.py

```
@@ -7,9 +7,9 @@
 INSTALL_PATH = os.getenv('NETALERTX_APP', '/app')
 sys.path.extend([f"{INSTALL_PATH}/front/plugins", f"{INSTALL_PATH}/server"])
 
-from helper import get_setting_value
-from utils.datetime_utils import timeNowDB
-from api_server.api_server_start import app
+from helper import get_setting_value   # noqa: E402 [flake8 lint suppression]
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)